velox/exec/tests/RowContainerTest.cpp

```diff
@@ -624,6 +624,81 @@ TEST_F(RowContainerTest, types) {
   EXPECT_LT(0, free.second);
 }
 
+TEST_F(RowContainerTest, extractNulls) {
+  constexpr int32_t kNumRows = 100;
+  auto batch = makeRowVector({
+      makeFlatVector<bool>(
+          kNumRows, [](auto row) { return row % 5 == 0; }, nullEvery(2)),
+      makeFlatVector<int8_t>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(3)),
+      makeFlatVector<int16_t>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(4)),
+      makeFlatVector<int32_t>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(5)),
+      makeFlatVector<int64_t>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(6)),
+      makeFlatVector<float>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(7)),
+      makeFlatVector<double>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(8)),
+      makeFlatVector<StringView>(
+          kNumRows, [](auto row) { return StringView("abcd"); }, nullEvery(9)),
+      BatchMaker::createVector<TypeKind::ARRAY>(
```

Comment:
BatchMaker is deprecated. Use VectorMaker or VectorFuzzer instead.

Reply:
I avoided BatchMaker for all other types but these since those methods were the most convenient. But I've now changed to APIs from VectorTestBase.h